bundles/automation/org.eclipse.smarthome.automation.module.script.defaultscope.test/ESH-INF/automation/jsr223/scopeFailure.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+var test = UnDefType.FOOBAR;
+if(test === undefined) {
+    throw "UnDefType.FOOBAR not defined";
+}
```

Comment:
missing newline

Reply:
please also add a simple failing test

```
var test2 = FOOBAR;
    if(test2 === undefined) {
    throw "FOOBAR not defined";
}
```

This may be combined with the existing one, simply `if (test === undefined && test2 === undefined) { throw...}`